### PR TITLE
Bump apollo-ci image to 0.3.56

### PR DESCRIPTION
**Pull request by janisz**
https://github.com/stackrox/rox-ci-image/releases/tag/0.3.56
https://github.com/openshift/release/pull/36092

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4648** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4648" target="_blank">Graphite</a> 
    * **PR #4658** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4658" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4658?utm_source=stack-comment).

**Review by porridge — COMMENTED** [review]
I assume this is still WiP and you don't intend to merge this with references to `-1-gd2a8e8bfde`?
Also, I think it would be better to call this PR `Bump apollo-ci image` to reflect better what's changed?

File: BUILD_IMAGE_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-stackrox-build-0.3.55
+stackrox-build-0.3.55-1-gd2a8e8bfde
```
Comment (porridge) [inline BUILD_IMAGE_VERSION:1]: Missed one place?

File: central/processlisteningonport/store/store.go [hunk 2]
```diff
@@ -8,6 +8,7 @@ import (
 )
 
 // Store provides storage functionality.
+//
```
Comment (porridge) [inline central/processlisteningonport/store/store.go:11]: Was this intended to go into this PR?

**Review by porridge — APPROVED** [review]
Tentative LGTM, modulo two questions inline.

File: central/processlisteningonport/store/store.go (hunk 2 above)
Comment (janisz) [reply central/processlisteningonport/store/store.go:11]: I missed this in #4648 `gofmt` has changed in 1.19

**Pull request merged**